packages/react/src/components/PortConfigurationModal/PortConfigurationModal.tsx

```diff
@@ -29,6 +29,10 @@ const PortConfigurationModalInternal: React.FC<Props> = ({ solution, visible, on
         type: data.portingLocation.value,
         workingDirectory: data.path
       };
+      if (portingLocation.workingDirectory.length > 260){
+        setError("path", "error", "The path length cannot exceed 260 characters. Please try a location that has a shorter path.");
+        return false;
+      }
```

Comment:
Define the same constant in the UI for the max path length in windows

Reply:
Addressed comments.